build/templates/functions.rst/unlock_session_wrapper.rst.mako

```diff
@@ -0,0 +1,11 @@
+<%page args="function, config, method_template, indent"/>\
+<%
+    import build.helper as helper
+%>\
+.. py:method:: ${function['python_name']}()
```

Comment:
If unlock_session became private, then this file is not needed, right? Same for lock_session_wrapper.rst.mako... ?